src/components/TextInputWithLabel.js

```diff
@@ -6,19 +6,20 @@ import styles from '../styles/styles';
 
 const propTypes = {
     /** Label text */
-    label: PropTypes.string.isRequired,
+    label: PropTypes.string,
 
     /** Text to show if there is an error */
     errorText: PropTypes.string,
 };
 
 const defaultProps = {
+    label: '',
     errorText: '',
 };
 
 const TextInputWithLabel = props => (
     <>
-        <Text style={[styles.formLabel]}>{props.label}</Text>
+        {!_.isEmpty(props.label) && <Text style={[styles.formLabel]}>{props.label}</Text>}
```

Comment:
NAB: Why are we making `props.label` optional? Seems strange to make a label optional on a component called `TextInputWithLabel`. I also don't see you using the `errorText` prop anywhere where you're using `TextInputWithLabel` without the `label` prop, so I'm wondering if this change is necessary.

Reply:
Sorta answered this in the other comment. But this component is likely going to change in the future when we apply some standardization to TextInputs across the app as it's kind of a mess. But we gotta keep rolling.